contexts/security-v3-unstable.jsonld

```diff
@@ -1,38 +1,92 @@
 {
-    "@context": [
-        {
-            "@version": 1.1
-        },
-        "https://w3id.org/security/v2",
-        {
-            "JsonWebSignature2020": {
-                "@id": "sec:JsonWebSignature2020"
-            },
-            "publicKeyJwk": {
-              "@id": "sec:publicKeyJwk",
-              "@type": "@json"
-            },
-            "ethereumAddress": {
-                "@id": "sec:ethereumAddress"
-            },
-            "publicKeyHex": {
-                "@id": "sec:publicKeyHex"
-            },
-            "blockchainAccountId":{
-                "@id": "sec:blockchainAccountId"
-            },
-            "Bls12381G1Key2020":{
-                "@id": "sec:Bls12381G1Key2020"
-            },
-            "Bls12381G2Key2020":{
-                "@id": "sec:Bls12381G2Key2020"
-            },
-            "BbsBlsSignature2020":{
-                "@id": "sec:BbsBlsSignature2020"
-            },
-            "BbsBlsSignatureProof2020":{
-                "@id": "sec:BbsBlsSignatureProof2020"
+  "@context": [
+    {
+      "@version": 1.1
+    },
+    "https://w3id.org/security/v2",
+    {
+      "JsonWebKey2020": {
+        "@id": "https://w3id.org/security#JsonWebKey2020"
```

Comment:
To be consistent with the syntax in previous security contexts, how about: `"JsonWebKey2020": "sec:JsonWebKey2020"`

Two benefits:
- uses the shorthand "sec" defined [here](https://w3c-ccg.github.io/security-vocab/contexts/security-v1.jsonld) instead of the full url "https://w3id.org/security#"
- more compact syntax with the implied "@id"

Thoughts?

Reply:
https://github.com/w3c-ccg/security-vocab/issues/57

its not documented anywhere yet, but I think we will be adding lint rules warning people to avoid them.